MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -0,0 +1,717 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal static class Sdl
+{
+    private const string NativeLibName = "SDL2.dll";
```

Comment:
> I found a good solution for deciding to load 32 or 64 bit dll.

I like that idea.  Keeps it simple.

> Maybe overriding the PATH environment variable would be fine?

That could be a backup.  The only change I would make would be to put the app path first before the existing PATH value.